ssl/record/methods/tls1_meth.c

```diff
@@ -650,6 +650,7 @@ struct record_functions_st tls_1_funcs = {
     tls_default_post_process_record,
     tls_get_max_records_multiblock,
     tls_write_records_multiblock, /* Defined in tls_multib.c */
+    tls_writev_records_default, /* TODO: Support multiblock */
```

Comment:
Don't submit TODO to repos unless they ask for it. Most of the time TODO stays forever in the code.